conn.go

```diff
@@ -386,12 +400,19 @@ func (conn *Conn) Send(msg *Message, ch chan *Call) *Call {
 		call.Method = iface + "." + member
 		call.Args = msg.Body
 		call.Done = ch
+		call.ctx = ctx
+		call.ctxCanceler = canceler
 		conn.calls.track(msg.serial, call)
+		go func() {
+			<-ctx.Done()
+			conn.calls.handleSendError(msg, ErrClosed)
```

Comment:
Do lines 408 and 411 cause duplicate calls to handleSendError or does that not matter the way this is used?